src/modules/sensors/sensor_params.c

```diff
@@ -43,10 +43,13 @@
  * Tube Pressure Drop
  * 		CAL_AIR_TUBED_MM: Diameter in mm of the pitot and tubes, must have the same diameter.
  * 		CAL_AIR_TUBELEN: Length of the tubes connecting the pitot to the sensor and the static + dynamic port length of the pitot.
+ * Venturi effect sensor
+ *    Supports TFSLOT airspeed sensor
```

Comment:
```suggestion
 *    Supports TFSLOT airspeed sensor
```

Is there a more google-able name than TFSLOT?

Reply:
We have prepared a documentation page that should improve gogleability of TFSLOT https://github.com/PX4/PX4-user_guide/blob/2a06420dd35cdeeacef266a8fe1d8e16c65e4cd5/en/sensor/airspeed_tfslot.md

Then the README expansion in the TFSLOT repository should also help.

Reply:
We have prepared a documentation page that should improve gogleability of TFSLOT https://github.com/PX4/PX4-user_guide/blob/2a06420dd35cdeeacef266a8fe1d8e16c65e4cd5/en/sensor/airspeed_tfslot.md

Then the README expansion in the TFSLOT repository should also help.

Reply:
The basic documentation for TFSLOT is published. After receiving this PR, I will expand it with settings with this profile. Is it possible to merge it?

Reply:
Hi, I have found [this TFSLOT documentation page](https://docs.px4.io/master/en/sensor/airspeed_tfslot.html) by using PX4 TFSLOT keywords now.  Is it acceptable to resolve that issue?